styles/modules/modals/_about.scss

```diff
@@ -0,0 +1,34 @@
+.about { 
+  .brandSmall {
+    width: 50px;
+    height: 50px;
+  }
+
+  .gemSmall {
+    width: 16px;
+    height: 16px;
+    margin: 1px;
+    background-size: contain;
+  }
+
+  #donation-qr-code {
+    width: 200px; 
+    height: 200px;
+    background: black;
+  } 
+
+  .columns {
```

Comment:
Creating column classes is a good idea. I think this would get used in other places, can you move it to _type.scss?

I think they should go in _type rather than _layout because the column properties applies to content.

Since there is no column width set in the column class, it will be one column, which isn't something we'd ever want. Can you set the column-count to 2 there, as the default? Then the twoColumn class isn't needed, and if we want to have other counts (like 3 or 4) we can create classes for those.

The li styles, however, are specific to the about modal, so they could stay here. I would give the container an about specific class for that, like contributorList.